docs/getting_started.md

```diff
@@ -82,68 +46,71 @@ $ sudo mv ./clusterctl.darwin_amd64 /usr/local/bin/clusterctl
+export VSPHERE_DATASTORE='DefaultDatastore'         # (required) The vSphere datastore to deploy the management cluster on
+export VSPHERE_NETWORK='vm-network-1'               # (required) The VM network to deploy the management cluster on
+export VSPHERE_RESOURCE_POOL='Resources'            # (required) The vSphere resource pool for your VMs
+export VSPHERE_FOLDER='vm'                          # (optional) The VM folder for your VMs, defaults to the root vSphere folder if not set.
+export VSPHERE_TEMPLATE='ubuntu-1804-kube-v1.13.6'  # (required) The VM template to use for your management cluster.
+export VSPHERE_DISK_GIB='50'                        # (optional) The VM Disk size in GB, defaults to 20 if not set
+export VSPHERE_NUM_CPUS='2'                         # (optional) The # of CPUs for control plane nodes in your management cluster, defaults to 2 if not set
+export VSPHERE_MEM_MIB='2048'                       # (optional) The memory (in MiB) for control plane nodes in your management cluster, defaults to 2048 if not set
+export SSH_AUTHORIZED_KEY='ssh-rsa AAAAB3N...'      # (optional) The public ssh authorized key on all machines in this cluster
 
 # Kubernetes configs
-export KUBERNETES_VERSION=1.13.6       # (optional) The Kubernetes version to use, defaults to 1.13.6
-export CLUSTER_NAME="my-cluster"       # (optional) The name for the management cluster, defaults to "capv-mgmt-example"
-export SERVICE_CIDR="100.64.0.0/13"    # (optional) The service CIDR of the management cluster, defaults to "100.64.0.0/13"
-export CLUSTER_CIDR="100.96.0.0/11"    # (optional) The cluster CIDR of the management cluster, defaults to "100.96.0.0/11"
+export KUBERNETES_VERSION='1.13.6'       # (optional) The Kubernetes version to use, defaults to 1.13.6
+export CLUSTER_NAME='my-cluster'       # (optional) The name for the management cluster, defaults to "capv-mgmt-example"
+export SERVICE_CIDR='100.64.0.0/13'    # (optional) The service CIDR of the management cluster, defaults to "100.64.0.0/13"
+export CLUSTER_CIDR='100.96.0.0/11'    # (optional) The cluster CIDR of the management cluster, defaults to "100.96.0.0/11"
+EOF
 ```
 
-With the above environment variables set, you can now run the generate yaml make target in the root directory of this project:
-```
-$ make prod-yaml
+With the above environment variable file it is now possible to generate the manifests needed to bootstrap the management cluster. The following command uses Docker to run an image that has all of the necessary templates and tools to generate the YAML manifests. Please note that the example mounts the current directory as the location where the YAML will be generated. Additionally, the `envvars.txt` file created above is mounted inside the the image in order to provide the generation routine with its default values:
+
+```shell
+$ docker run --rm \
+  -v "$(pwd)/my-cluster":/out \
+  -v "$(pwd)/envvars.txt":/out/envvars.txt:ro \
```

Comment:
This approach was used to load the env vars since a Docker env var file doesn't allow for the verbose commenting provided by the original author of the env var examples above. This was a hack to preserve the original annotations.